.github/workflows/release.yml

```diff
@@ -15,10 +15,16 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - run: >-
-          aws s3 cp
-          s3://${{ env.BUCKET }}/${{ env.PREFIX }}/version/${{ github.event.inputs.version }}/${{ env.SRC_FILE }}
-          s3://${{ env.BUCKET }}/${{ env.PREFIX }}/index.html
+      - run: |-
```

Comment:
I think the existing "chomp" (`>-`) method is easier to read than the new version which requires the `\` line continuation character but that is just personal taste.